decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super
```

Comment:
- [ ] Kindly make sure that the program is working properly, currently, when I try to run the app, the following error appears:
![image](https://user-images.githubusercontent.com/62290545/210416196-d4c0768c-de5c-4245-8f62-a47ee4d9fb10.png)

Reply:
Hello @ShadyShawkat, thanks for your review, I have a question about this error because when I remove the `super` from that file everything runs perfectly but the linters throw an error.